cms/search/tests/test_kafka_integration.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+import time
+
+from django.test import TestCase, override_settings
+from kafka import KafkaConsumer
+
+from cms.search.publishers import KafkaPublisher
+from cms.standard_pages.tests.factories import InformationPageFactory
+
+
+@override_settings(
+    KAFKA_SERVER="localhost:9092",
+    KAFKA_TOPIC_CREATED_OR_UPDATED="search-content-updated",
+    KAFKA_TOPIC_DELETED="search-content-deleted",
+)
+class KafkaIntegrationTests(TestCase):
+    """These tests will attempt to connect to a real Kafka instance at localhost:9092."""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Create a KafkaConsumer to listen to the same topics
+        # so we can verify messages that come in.
+        cls.consumer_created = KafkaConsumer(
+            "search-content-updated",
+            bootstrap_servers=["localhost:9092"],
+            auto_offset_reset="earliest",
+            enable_auto_commit=True,
+            group_id="test-group-created-updated",
+        )
+
+        cls.consumer_deleted = KafkaConsumer(
+            "search-content-deleted",
+            bootstrap_servers=["localhost:9092"],
+            auto_offset_reset="earliest",
+            enable_auto_commit=True,
+            group_id="test-group-deleted",
+        )
+
+        cls.publisher = KafkaPublisher()
+        logging.getLogger(__name__).info("KafkaIntegrationTests: setUpClass complete.")
+
+    @classmethod
+    def tearDownClass(cls):
+        # Close the consumers
+        cls.consumer_created.close()
+        cls.consumer_deleted.close()
+        super().tearDownClass()
+
+    def _poll_for_message(self, consumer, expected_uri, timeout_secs=5):
```

Comment:
```suggestion
    def _poll_for_message(self, consumer, expected_uri, timeout_seconds=5):
```

let's be more verbose here

Reply:
Roger, understood! Done!